server.js

```diff
@@ -0,0 +1,23 @@
+const express = require('express');
+const cors = require('cors');
+const axios = require('axios');
+require('dotenv').config();
+
+const app = express();
+app.use(express.json());
+app.use(cors({ origin: '*' }));
+
+const AUTH_CREDENTIALS = {
+  user: process.env.AUTH_LOGIN,
+  password: process.env.AUTH_PASSWORD
+};
+
+app.get('/token', (req, res) => {
```

Comment:
Suggest to have all the strings in a separate module as constants

Reply:
Ok. fixed. Thanks :)